build is failing
```
======================================================================
FAIL: testBasicChunking (corehq.apps.case_importer_v1.tests.ImporterTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/mnt/commcare-hq-ro/corehq/apps/case_importer_v1/tests.py", line 318, in testBasicChunking
    res = do_import(file, config, self.domain, chunksize=2)
  File "/mnt/commcare-hq-ro/corehq/apps/case_importer_v1/tasks.py", line 278, in do_import
    _submit_caseblocks(domain, config.case_type, caseblocks)
  File "/mnt/commcare-hq-ro/corehq/apps/case_importer_v1/tasks.py", line 97, in _submit_caseblocks
    '({}): {}'.format(domain, ", ".join(properties))
  File "/mnt/commcare-hq-ro/corehq/util/soft_assert/core.py", line 33, in __call__
    return self._call(assertion, msg, obj)
  File "/mnt/commcare-hq-ro/corehq/util/soft_assert/core.py", line 47, in _call
    raise AssertionError(msg)
AssertionError: error adding inferred export properties in domain (importer-test): location, age, sex
```

--- a/corehq/apps/case_importer_v1/tasks.py
+++ b/corehq/apps/case_importer_v1/tasks.py
@@ -90,7 +90,7 @@ def do_import(spreadsheet, config, domain, task=None, chunksize=CASEBLOCK_CHUNKS
                         case_type,
                         properties,
                     )
-                else:
+                elif len(properties):
                     _soft_assert = soft_assert(notify_admins=True)
                     soft_assert(
                         False,

--- a/corehq/apps/case_importer_v1/tasks.py
+++ b/corehq/apps/case_importer_v1/tasks.py
@@ -13,6 +13,7 @@ from corehq.form_processor.interfaces.dbaccessors import CaseAccessors
 from soil import DownloadBase
 from dimagi.utils.prime_views import prime_views
 from couchdbkit.exceptions import ResourceNotFound
+from corehq.util.soft_assert import soft_assert
 import uuid
 
 POOL_SIZE = 10
@@ -82,12 +83,20 @@ def do_import(spreadsheet, config, domain, task=None, chunksize=CASEBLOCK_CHUNKS
                 )
             else:
                 properties = set().union(*map(lambda c: set(c.dynamic_case_properties().keys()), cases))
-                add_inferred_export_properties.delay(
-                    'CaseImporter',
-                    domain,
-                    case_type,
-                    properties,
-                )
+                if case_type and len(properties):
+                    add_inferred_export_properties.delay(
+                        'CaseImporter',
+                        domain,
+                        case_type,
+                        properties,
+                    )
+                else:
+                    _soft_assert = soft_assert(notify_admins=True)
+                    soft_assert(
+                        False,
+                        'error adding inferred export properties in domain '
+                        '({}): {}'.format(domain, ", ".join(properties))
+                    )
         return err
 
     for i in range(row_count):

--- a/corehq/apps/case_importer_v1/tasks.py
+++ b/corehq/apps/case_importer_v1/tasks.py
@@ -90,13 +90,12 @@ def do_import(spreadsheet, config, domain, task=None, chunksize=CASEBLOCK_CHUNKS
                         case_type,
                         properties,
                     )
-                elif len(properties):
-                    _soft_assert = soft_assert(notify_admins=True)
-                    soft_assert(
-                        False,
-                        'error adding inferred export properties in domain '
-                        '({}): {}'.format(domain, ", ".join(properties))
-                    )
+                _soft_assert = soft_assert(notify_admins=True)
+                _soft_assert(
+                    len(properties) == 0 and not case_type,
+                    'error adding inferred export properties in domain '
+                    '({}): {}'.format(domain, ", ".join(properties))
+                )
         return err
 
     for i in range(row_count):

